.circleci/config.yml

```diff
@@ -103,6 +103,29 @@ jobs:
           environment:
             RUST_BACKTRACE: 1
           command: cargo test --verbose --frozen
+    
+  test_mockchain_all_features:
```

Comment:
This may become useful for testing other crates with non-default features, so I'd name the job `test_all_features` with intent to add more per-crate build/test steps as needed later.